packages/web-components/src/components/va-file-input/va-file-input-upgrader.ts

```diff
@@ -300,17 +301,16 @@ const removeOldPreviews = (dropTarget, instructions) => {
  * @param {Object} fileNames - The selected files found in the fileList object.
  * @param {Array} fileStore - The array of uploaded file names created from the fileNames object.
  */
-const updateStatusMessage = (statusElement, fileNames, fileStore) => {
+const updateStatusMessage = (statusElement: HTMLDivElement | Element, fileNames: FileList | File[], fileStore: string[]): void => {
```

Comment:
This can be simplified to `statusElement: Element` as every element will match that type.